### DISCUSSION TITLE
Keep Project Status (runing, debug, ....)

### DISCUSSION THREAD

**Discussion by SuperService** [alefragnani/vscode-project-manager#616] [Q&A] [answered]
vscode-project-manager is awesome for our developer
thanks 👏
I have to manage serveral project in dev status
etc
first project is runing in vscode ( http://localhost:3000 )
and then I go to other project with vscode-project-manager
and second project is runing (http://localhost:4000)

At that time if I go to again first project, project is not runing ( like open folder )

Could you keep project status?
thanks

**Comment by alefragnani** [comment, marked as answer]
Hi @SuperService ,

Thanks for your kind words. I'm glad the extension is useful for your developers.

When you say _first project is _running in vscode_, I guess you started some kind of Terminal or other extension that runs your project, right? And about _go to other project_ I suppose you are switching from the first to the second project using Project Extension commands, right?

If that's the case, what you need a _session restore_ feature. Something that returns the VS Code window to the state you left in the last session.

Unfortunately, the Project Manager extension is not supposed to do that. But you are not alone. VS Code itself has made a few improvements in the latest versions to restore _some states_ of the Terminal, so you could take a look if those improvements helps you. If you are not using Terminal to run the projects, but instead, some specific extension, I suggest you to open an issue on that extension, in order to properly restore the state you were before you left the session.

Hope this helps.

**Reply by SuperService** [reply]
thanks for your understand
so my solution is vs code extension problem
have a good day
